app/controllers/admin/users_controller.rb

```diff
@@ -2,12 +2,12 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     authorize(User)
-    @users = policy_scope(User.member).page(params[:page])
+    @users = sort(policy_scope(User.member).page(params[:page]))
```

Comment:
I feel like I'm missing something. Don't you need to do the sort before you do the pagination?